src/assets/src/components/meetingType.tsx

```diff
@@ -52,14 +54,14 @@ interface BackendSelectorProps {
 }
 
 export const BackendSelector: React.FC<BackendSelectorProps> = (props) => {  
-    const options = Array.from(props.allowedBackends)
-        .map(
-            a => (
-                <option key={a} value={a}>
-                    {getBackendByName(a as EnabledBackendName, props.backends).friendly_name}
-                </option>
-            )
+    const enabledAllowedBackends = props.backends.filter(
+        (b) => b.enabled && props.allowedBackends.has(b.name)
+    );
+
+    const options = enabledAllowedBackends.map(
+        (a) => <option key={a.name} value={a.name}>{a.friendly_name}</option>
     );
+
```

Comment:
I added this change recently. Prior to the `BackendPhaser.phase_out` being called, you could still see BlueJeans as an option in the dropdown, though the API still threw errors appropriately. I suppose we could debate whether all these changes to `meetingType.tsx` are necessary if we plan to use the DB migration code immediately.